android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/utils/CallbackMessages.java

```diff
@@ -0,0 +1,8 @@
+package covidsafepaths.bt.exposurenotifications.utils;
+
+public class CallbackMessages {
```

Comment:
whats the plan with these guys? do we need to be handling these cases on the JS side?

Reply:
The idea is to have one place on the Android native layer that shows all the messages being bubbled up to the js layer. `String+Extensions.swift` on the iOS side I think would be the equivalent, but correct me if I'm wrong on that.

Right now I don't think we need to handle them on the JS side. For the `requestExposureNotificationAuthorization` I'm replicating iOS which sends back a `genericSuccess` message or an error string.

Reply:
got it, thanks for the context, makes a lot of sense to me.